pandas/core/series.py

```diff
@@ -3687,7 +3687,9 @@ def f(x):
 
         if len(mapped) and isinstance(mapped[0], Series):
             from pandas.core.frame import DataFrame
-            return DataFrame(mapped.tolist(), index=self.index)
+            # GH 25959 use pd.array instead of tolist
+            # so extension arrays can be used
```

Comment:
Sorry,can mapped be 2d here? If so, we shouldn’t use pd.array

Reply:
Yes, it is possible for it to be a nested series.